### DISCUSSION TITLE
Flexible assessment tool

### DISCUSSION THREAD

**Discussion by pglevy** [Bixal/rapid-response-team#17] [Archive]
Seems common in government to ask a series of questions and then calculate a response. It would have to be standalone, not requiring integration with data systems. (Or maybe that something connecting our own data set.) But maybe some version of this that is somewhat customizable for different purposes. For example: https://businessrelief.usdigitalresponse.org/

**Comment by ghost** [comment]
I think this one would be a great starting project to jump start the RR team and get our minds rolling

**Reply by ghost** [reply]
- be specific on use cases so we dont go down a rabbit hole
- what specific question needs answered by many?

**Comment by pglevy** [comment]
Future versions of this should build on [USWDS Screener Prototyping Tool](https://github.com/Bixal/uswds-screener-prototyping-tool)
